test/unit/services/DeclarationServiceSpec.scala

```diff
@@ -37,6 +37,7 @@ import uk.gov.hmrc.customs.declaration.xml.MdgPayloadDecorator
 import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
 import uk.gov.hmrc.play.test.UnitSpec
 import util.ApiSubscriptionFieldsTestData._
+import util.TestData
```

Comment:
This is not needed. if you removed this and "TestData." from line 56 you can remove this